anim.js

```diff
@@ -8,9 +8,9 @@ class Anim {
   }
 
   add(to, duration = resolution, options = {}) {
-    options.easing = options.easing || 'linear';
+    options.easing = options.easing || "linear";
```

Comment:
Our `eslint.rc` contains the rule `"quotes": [2, "single", "avoid-escape"]` but your code formatting converts single quotes to double quotes.
Your formatting should consider our linting rules.